realm/realm-library/src/androidTest/java/io/realm/RealmAsyncQueryTests.java

```diff
@@ -223,7 +288,7 @@ public void execute(Realm realm) {
     @RunTestInLooperThread
     public void executeTransaction_async_realmClosedOnError() {
         final AtomicInteger counter = new AtomicInteger(100);
-        final Realm realm = Realm.getInstance(looperThread.createConfiguration("testClosedBeforeAsyncTransactionSuccess"));
+        final Realm realm = Realm.getInstance(looperThread.createConfiguration("executeTransaction_async_realmClosedOnError"));
```

Comment:
No need to create a specific named Realm anymore, just use `looperThread.realm`?